src/core/css/SearchResult.scss

```diff
@@ -32,12 +32,18 @@
   font-size: $font-size-default;
   font-weight: bold;
   margin: 0;
+  min-width: 1%;
```

Comment:
Reference:
https://stackoverflow.com/questions/12022288/how-to-keep-a-flex-item-from-overflowing-due-to-its-text#answer-12131365